plugins/tabletools/plugin.js

```diff
@@ -227,14 +227,17 @@
 		return null;
 	}
 
-	function getCellColIndex( cell, isStart ) {
+	function getCellColIndex( cell ) {
```

Comment:
In the meantime, `tableselection` plugin have been created which also utilizes this function (passing `isStart=true`) in two places. The code there should be also adjusted. All `plugins/table*` unit tests are still passing which looks promising.